python/ray/train/callbacks/logging.py

```diff
@@ -314,6 +323,10 @@ def handle_result(self, results: List[Dict], **info):
 
     def finish_training(self, error: bool = False, **info):
         self._file_writer.close()
+        logger.info(
+            "To view the training results in your browser, run `tensorboard --logdir %s`"
+            % self.logdir
```

Comment:
nit: Would recommend using f-strings instead!
```suggestion
            f"To view the training results in your browser, run `tensorboard --logdir {self.logdir}`"
```

Reply:
Yes

Reply:
Yes